vllm/utils.py

```diff
@@ -178,7 +178,7 @@ def get_ip() -> str:
     except OSError:
         # try ipv6
         s = socket.socket(socket.AF_INET6, socket.SOCK_DGRAM)
-        s.connect(("dns.google", 80))
+        s.connect(("2001:4860:4860:0:0:0:0:8888", 80))
```

Comment:
FYI: https://developers.google.com/speed/public-dns/docs/using#addresses

Reply:
@hliuca Why do we need this change?

Reply:
Hi, @WoosukKwon. `dns.google` use DoT, and maybe we should avoid DoT here.

https://developers.google.com/speed/public-dns/docs/dns-over-tls

Reply:
Got it. Maybe you can split it into another PR. @hliuca

Reply:
@WoosukKwon I agree with you. I'm not the authors of the PR and just commented on the "irrelevant" changing.

@hliuca Are you still working on this PR? Could you move this changing into another PR?

Reply:
@chenxu2048 I see. I thought you were the author. Sorry for that. 🤦